micro-ui/web/micro-ui-internals/packages/modules/sandbox/src/pages/employee/ProductPage/ProductPageComponent.js

```
@@ -24,15 +24,17 @@ const ProductsPageComponent = ({detailsConfig}) => {
             <div className="product-header">
-              {Digit.Utils.iconRender(product.icon,"#c84c0e")}
+            <div className="icon-wrap">
+              {Digit.Utils.iconRender(product.icon, "#c84c0e")}
+            </div>
               <HeaderComponent className="product-title">{t(product.heading)}</HeaderComponent>
             </div>
             <CardText className="product-description">{t(product?.cardDescription)}</CardText>
             <Button
               className="explore-button"
               variation="secondary"
               label={t("COMMON_EXPLORE") + " →"}
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)